geojson2svg/geojson2svg.go

```diff
@@ -286,49 +301,42 @@ func drawLineString(sf ScaleFunc, w io.Writer, ps [][]float64, attributes string
 		fmt.Fprintf(path, "%f %f,", x, y)
 	}
 	endTag := endTag("path", title)
-	fmt.Fprintf(w, `%s<path d="%s"%s%s`, newline, trim(path), attributes, endTag)
+	fmt.Fprintf(w, `<path d="%s"%s%s`, strings.TrimSuffix(path.String(), ","), attributes, endTag)
 }
 
 func drawMultiLineString(sf ScaleFunc, w io.Writer, pps [][][]float64, attributes string, title string) {
```

Comment:
worth thinking about more verbose variable names - e.g. `pps` means nothing to me